### ISSUE TITLE
Global Error Type

### ISSUE THREAD

**Issue by inceenes10** [HuolalaTech/react-query-kit#50]
I couldn't make global error type in `react-query-kit`. I have an code block like below that works for @tanstack/react-query but not works with react-query.

```
import "@tanstack/react-query";

declare module "@tanstack/react-query" {
	interface Register {
		defaultError: IError;
	}
}

```

when I call a query with this configuration, the error defined below is `IError` type.

```
const { error} = useQuery({ 
    ... 
})
```

How can I make the same effect for react-query-kit

**Comment by liaoliao666** [comment]
This works for me. Is your ReactQuery version v5?

**Comment by inceenes10** [comment]
yes, it is v5

**Comment by inceenes10** [comment]
yes, it is v5

**Comment by liaoliao666** [comment]
Do you put this code in entry file? If not can you provide me a sample?

**Comment by inceenes10** [comment]
I put this code types directory in the project

**Comment by liaoliao666** [comment]
You should put this code in app.tsx i think.

**Comment by inceenes10** [comment]
I use app directory, that's why I don't have app.tsx

**Comment by liaoliao666** [comment]
It should have an entry file called something like app.tsx or main.tsx whatever and put this code in it.

**Comment by inceenes10** [comment]
It does not work

**Comment by inceenes10** [comment]
I put the code in page.tsx and layout.tsx. It does not work

**Comment by liaoliao666** [comment]
From my research it doesn't work in next js. It will be fixed in the next version as soon as possible.

**Comment by inceenes10** [comment]
Thank you for your interest. Is there anything that I contribute?

**Comment by liaoliao666** [comment]
Thanks, It already fixed in v3.2.2

**Comment by inceenes10** [comment]
Thank you
